frontend/src/pages/FactQuizQuestionPage.jsx

```diff
@@ -166,7 +166,7 @@ export const FactQuizQuestionPage = () => {
                                                 <Button
                                                     variant="contained"
                                                     color="primary"
-                                                    onClick={handleAnswer}
+                                                    href="/tietovisa/yhteenveto"
```

Comment:
If I'm reading this correctly, the `yhteenveto` page has no way to submit the quiz as of now so I would revert this line until that functionality is implemented.

```suggestion
                                                    onClick={handleAnswer}
```